sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_delete_test_live.cpp

```diff
@@ -145,10 +149,11 @@ TEST_F(KeyVaultClientTest, DoubleDelete)
         = keyClient.CreateKey(keyName, Azure::Security::KeyVault::Keys::KeyVaultKeyType::Ec);
   }
   {
-    auto duration = std::chrono::system_clock::now() + std::chrono::minutes(3);
+    auto duration
+        = std::chrono::system_clock::now() + std::chrono::minutes(testPollingTimeOutMinutes);
     auto cancelToken = Azure::Core::Context::ApplicationContext.WithDeadline(duration);
     auto keyResponseLRO = keyClient.StartDeleteKey(keyName);
-    auto keyResponse = keyResponseLRO.PollUntilDone(std::chrono::milliseconds(1000), cancelToken);
+    auto keyResponse = keyResponseLRO.PollUntilDone(std::chrono::seconds(60), cancelToken);
```

Comment:
Wouldn't polling every 60 seconds make the tests really slow, especially if it finishes quickly? Maybe we can change it to 10 seconds instead?

Reply:
The less we poll, the better for the service and the billing :).  We use 2 min polling in some other tests.

Reply:
Makes sense. Hopefully this fixes the intermittent test issue :)